make-rules/shared-macros.mk

```diff
@@ -170,40 +185,104 @@ PROTOUSRSHAREMAN1MDIR =	$(PROTO_DIR)/$(USRSHAREMAN1MDIR)
 PROTOUSRSHAREMAN3DIR =	$(PROTO_DIR)/$(USRSHAREMAN3DIR)
 PROTOUSRSHAREMAN4DIR =	$(PROTO_DIR)/$(USRSHAREMAN4DIR)
 PROTOUSRSHAREMAN5DIR =	$(PROTO_DIR)/$(USRSHAREMAN5DIR)
+PROTOUSRSHAREMAN8DIR =	$(PROTO_DIR)/$(USRSHAREMAN8DIR)
 PROTOUSRSHARELOCALEDIR =	$(PROTO_DIR)/$(USRSHARELOCALEDIR)
-PROTOUSRKERNELDRVDIR =	$(PROTO_DIR)/$(USRKERNELDRVDIR)
+PROTOUSRKERNELDRVDIR =  	$(PROTO_DIR)/$(USRKERNELDRVDIR)
+PROTOUSRKERNELDRVDIR32 =	$(PROTO_DIR)/$(USRKERNELDRVDIR32)
 PROTOUSRKERNELDRVDIR64 =	$(PROTO_DIR)/$(USRKERNELDRVDIR64)
 
+PROTOUSRBINDIR.32 = 	$(PROTOUSRBINDIR)
+PROTOUSRBINDIR.64 = 	$(PROTOUSRBINDIR64)
+PROTOUSRSBINDIR.32 =	$(PROTOUSRSBINDIR)
+PROTOUSRSBINDIR.64 =	$(PROTOUSRSBINDIR64)
+PROTOUSRLIBDIR.32 = 	$(PROTOUSRLIBDIR)
+PROTOUSRLIBDIR.64 = 	$(PROTOUSRLIBDIR64)
 
 SFWBIN =	/usr/sfw/bin
+SFWBIN32 =	$(SFWBIN)/$(MACH32)
```

Comment:
we don't have or use /usr/sfw/bin/$(MACH)
/usr/sfw/bin is just a historic artefact, containing links.